deck-o-cards.js

```diff
@@ -10,13 +10,27 @@ function deck_o_cards() {
   // Make 52 card objects and store them in the "cards" array
   // Hint: use 2 for loops
 
+  var topCard = null;
```

Comment:
Rather than creating a variable just so it's null, I would create AND set at the same time on line 27.